setup.py

```diff
@@ -284,4 +322,7 @@ def _make_strs(paths: List[Union[str, Path]]) -> List[str]:
         cmdclass={"build_ext": BuildBases},
         options={"install": {"optimize": 1}},
         ext_modules=extensions,
+        package_data={'cx_Freeze':
```

Comment:
If you use MANIFEST.in, you can remove the changes in setup.py

Reply:
I've never used MANIFEST.in before, thanks for pointing that out.

Reply:
after
recursive-include cx_Freeze/hooks *.conf *.sh *.qrc
add
recursive-include cx_Freeze/initscripts *.txt